torch/nn/parallel/_functions.pyi

```diff
@@ -0,0 +1,4 @@
+import types
+from typing import Tuple
+
+def _get_device_impl() -> Tuple[str, types.ModuleType]: ...
```

Comment:
Why this pyi file is needed?

Reply:
We followed the suggestions from @kumpera , and add a stub version of the module for type hints. Any rules for these pyi files?

Reply:
A pyi file is only to be ised for Python functions from C extension. Everything else should be annotated inline